### PR TITLE
Added Test for Ordered Imports

### PR DESCRIPTION
**Pull request by carlburgess**
Added test for ordered imports using  static,  com.sun.  , org.omg,  org.w3c  java, javax

### CODE REVIEW

File: test/mergeTestsTest_data/import_ordering/existing_imports/ordered_imports/T8.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.Timeout;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
```
Comment (justincyy) [inline test/mergeTestsTest_data/import_ordering/existing_imports/ordered_imports/T8.java]: The org.* and project imports should be in the same block. The filtering for the JDK block is done by seeing whether it starts with java or javax.

File: test/mergeTestsTest_data/import_ordering/existing_imports/ordered_imports/T3.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import static diffblue.AddInt.sum;
+import static diffblue.Sun.land;
+import static java.lang.Math.PI;
+
+import diffblue.RedGreenBlue.group;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+
```
Comment (justincyy) [inline test/mergeTestsTest_data/import_ordering/existing_imports/ordered_imports/T3.java]: org.* and diffblue.* should be in the same block.

File: test/mergeTestsTest_data/import_ordering/existing_imports/ordered_imports/T2.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import static org.crb.floatmethod.AddFloat.sumf;
+import static org.junit.Assert;
+
+import com.diffblue.check;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.Timeout;
+
```
Comment (justincyy) [inline test/mergeTestsTest_data/import_ordering/existing_imports/ordered_imports/T2.java]: org.* and com.* should be in the same block.

Comment (carlburgess) [reply test/mergeTestsTest_data/import_ordering/existing_imports/ordered_imports/T2.java]: Fixed

File: test/mergeTestsTest_data/import_ordering/existing_imports/ordered_imports/T3.java (hunk 2 above)
Comment (carlburgess) [reply test/mergeTestsTest_data/import_ordering/existing_imports/ordered_imports/T3.java]: Fixed

File: test/mergeTestsTest_data/import_ordering/existing_imports/ordered_imports/T8.java (hunk 1 above)
Comment (carlburgess) [reply test/mergeTestsTest_data/import_ordering/existing_imports/ordered_imports/T8.java]: Fixed

**Pull request merged**